pkg/controller/pv.go

```diff
@@ -81,17 +85,19 @@ func (r *reconcilePV) Reconcile(ctx context.Context, request reconcile.Request)
 			klog.V(5).Infof("failed to get AzVolume (%s): %v", diskName, err)
 			return reconcile.Result{Requeue: true}, err
 		}
-		r.retryMutex.RLock()
-		numRetry, ok := r.retryMap[azVolumeName]
-		r.retryMutex.RUnlock()
-		if !ok {
-			r.retryMutex.Lock()
-			if _, ok = r.retryMap[azVolumeName]; !ok {
+
+		v, ok := r.retryMap.get(azVolumeName)
+		var numRetry *uint32
```

Comment:
Why are we retrying here and not rescheduling for the next reconciler cycle?

Reply:
We are rescheduling and requeuing the request but we are limiting the number of requeues.